python/whylogs/core/model_performance_metrics/confusion_matrix.py

```diff
@@ -60,8 +82,6 @@ def __init__(
                     " selectively log the most important labels or configure the threshold of"
                     " {MODEL_METRICS_MAX_LABELS} higher by setting MODEL_METRICS_MAX_LABELS."
                 )
-
-            self.labels = sorted(labels)
```

Comment:
why is it sage to remove the sort here?

Reply:
because the sorted is done above in the if/else (lines 66 and 69)

Reply:
I see it now, thanks!